config.py

```diff
@@ -90,29 +90,17 @@
 DIVERT_TO_PRIVATE = ('help', )
 
 ROOMS_TO_JOIN = (
+    'announce',
     'coala',
-    'coala/offtopic',
-    'cobot-test',
-    'corobo',
```

Comment:
these names are all valid names, and they are used with the Gitter backend, for our Gitter community at https://gitter.im/coala/

This issue is about also mapping them correctly to our Zulip community, at https://coala.zulipchat.com/ , which you will need to join to see the list of rooms available there , to work out the mapping .

Especially see `if BACKEND == 'Gitter':` below.  This task is about adding `if BACKEND == 'Zulip':` where most of the work needs to be done for this task.